### PR TITLE
HPCC-18964 Add translations to Login.html

### PR DESCRIPTION
**Pull request by miguelvazq**
Signed-off-by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: esp/src/lws.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const debugServerIP = "192.168.99.103";
+const debugServerIP = "10.176.152.166";
```
Comment (richardkchapman) [inline esp/src/lws.config.js]: Doesn't deel like information that should be checked in (even if it's an internal IP)

Comment (miguelvazq) [reply esp/src/lws.config.js]: Agreed will remove and fix up the commit message as well.

**Comment by miguelvazq** [comment]
@GordonSmith Please review. @kunalaswani FYI seems there was some translations that missed the cut.

**Review by GordonSmith — APPROVED** [review]
Looks fine.

**Comment by miguelvazq** [comment]
@richardkchapman ready to merge.

**Comment by richardkchapman** [comment]
Do we need to add anything to the recently-done translation files?

**Comment by GordonSmith** [comment]
@kunalaswani - they will need another iteration for new words (and will default to English in the meantime)

**Comment by kunalaswani** [comment]
@richardkchapman once all of them are in i will need to do one more run for words that have been added last min yes. I can open a child ticket if you think that is the best way to go about it.

**Pull request merged**